Gemfile

```diff
@@ -17,8 +17,8 @@ gem 'sass-rails', '~> 5.0'
 # Use Uglifier as compressor for JavaScript assets
 gem 'uglifier', '>= 1.3.0'
 # See https://github.com/rails/execjs#readme for more supported runtimes
-# gem 'therubyracer', platforms: :ruby
-
+# gem 'i18n', platforms: :ruby
+gem "i18n-js"
 # Use CoffeeScript for .coffee assets and views
 gem 'coffee-rails', '~> 4.2'
```

Comment:
Toàn bộ file rubu đều dùng nháy đôi hết nhé. E chỉnh sửa nguyên file này lun nhé